website/_redirects

```diff
@@ -15,6 +15,7 @@
 /intro/getting-started/next-steps.html  https://learn.hashicorp.com/nomad/getting-started/next-steps
 /intro/getting-started/next-steps       https://learn.hashicorp.com/nomad/getting-started/next-steps
 
+/guides/load-balancing                  https://learn.hashicorp.com/nomad?track=load-balancing#operations-and-development
```

Comment:
These are all missing `.html` versions. May be worth checking whether there are many inbound links to the `.html` extension on these pages and adding another redirect if so